public/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "OwnTracks Frontend",
+    "description": "OwnTracks Frontend",
+    "icons": [
+      {
+        "src": "icon-180x180.png",
+        "type": "image/png",
+        "sizes": "180x180"
+      },
+      {
+        "src": "OwnTracks.svg",
+        "sizes": "any"
+      }
+    ],
+    "start_url": ".",
+    "background_color": "#3f51b5",
+    "display": "standalone",
+    "scope": ".",
+    "theme_color": "#3f51b5"
+  }
```

Comment:
Missing newline at end of file and the indentation looks a bit off, please run `prettier` on this.